### PR TITLE
[@types/fibers] Upgrade types for node-fibers

### PR DESCRIPTION
**Pull request by smac89**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/laverdet/node-fibers#api-documentation
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

File: types/fibers/index.d.ts [hunk 1]
```diff
@@ -1,23 +1,91 @@
-// Type definitions for fibers
+// Type definitions for node-fibers 3.1
```
Comment (uniqueiniquity) [inline types/fibers/index.d.ts]: It looks like the lint rules for this package are disabled. Could you at least remove the line in the `tslint.json` file in this folder that says `dt-header: false` and fix any errors that come up? You are welcome to remove any other such line from the `tslint.json` file, though that might be a lot of work depending on what issues are being ignored. Thanks!

**Review by uniqueiniquity — CHANGES_REQUESTED** [review]

File: types/fibers/index.d.ts (hunk 1 above)
Comment (smac89) [reply types/fibers/index.d.ts]: I believe I have no enabled the linting

**Review by uniqueiniquity — APPROVED** [review]

**Comment by smac89** [comment]
Bump. Can we get this merged?
Thank you

**Comment by uniqueiniquity** [comment]
Sorry for the delay!

**Pull request merged**